testdata/terraform/cloudflare_zone_settings_override/test.tf

```diff
@@ -9,7 +9,7 @@ resource "cloudflare_zone_settings_override" "terraform_managed_resource" {
     browser_check             = "on"
     cache_level               = "aggressive"
     challenge_ttl             = 2700
-    ciphers                   = ["ECDHE-RSA-AES128-GCM-SHA256", "AES128-SHA"]
```

Comment:
i don't think we can blindly sort these as they are in the order at which they should be offered iirc.